BREEZE.rst

```diff
@@ -46,409 +46,552 @@ Here is the short 10 minute video about Airflow Breeze
 Prerequisites
 =============
 
-Docker
-------
+Docker Community Edition
+-----------------------
 
-You need latest stable Docker Community Edition installed and on the PATH. It should be
-configured to be able to run ``docker`` commands directly and not only via root user. Your user
-should be in the ``docker`` group. See `Docker installation guide <https://docs.docker.com/install/>`_
+- **Version**: Install the latest stable Docker Community Edition and add it to the PATH.
+- **Permissions**: Configure to run the ``docker`` commands directly and not only via root user. Your user should be in the ``docker`` group. See `Docker installation guide <https://docs.docker.com/install/>`_ for details.
+- **Disk space**: On macOS, increase your available disk space before starting to work with the environment. At least 128 GB of free disk space is recommended. You can also get by with a smaller space but make sure to clean up the Docker disk space periodically. See also `Docker for Mac - Space <https://docs.docker.com/docker-for-mac/space>`_ for details on increasing disk space available for Docker on Mac.
```

Comment:
We can add a ref to https://docs.docker.com/config/pruning/ for cleaning up unused "things" :)

Reply:
Yep